ansible/roles/uan_disk_config/defaults/main.yml

```
@@ -31,9 +31,13 @@
 uan_require_disk: false
 
 uan_device_name_filter: "^sd[a-f]$"
-uan_device_host_filter: ""
-uan_device_model_filter: ""
-uan_device_vendor_filter: ""
+uan_device_name_exclude_filter: ""
+uan_device_host_filter: []
+uan_device_host_exclude_filter: []
+uan_device_model_filter: []
+uan_device_model_exclude_filter: []
+uan_device_vendor_filter: []
+uan_device_vendor_exclude_filter: ["LIO-ORG"]
```

Since these are really lists, can we have it in the form:
```suggestion
uan_device_vendor_exclude_filter:
  - "LIO-ORG"
```

---

Changed as recommended.